### ISSUE TITLE
Enhancement request: A `--version` switch would be useful

### ISSUE THREAD

**Issue by kseistrup** [bwhmather/ssort#110]
This enhancement request is valid as of commit 018ca2d1a499f4a5074629eaaa95f0c7f2134f1c

Please provide a `-V`/`--version` switch that prints the executable's name and version (and possibly version date) to standard output and exits successfully.

E.g.:

```sh
» ssort --version
ssort v0.12.3 (2024-01-21)
#              ^^^^^^^^^^
#               ISO 8601
```

Such a switch is useful for:

+ users or admins who want to know if an upgrade is needed, or when
+ referring to a given version when submitting a bug report (the submitter may not be the one who installed the binary, and may nbot be able to refer to a commit).

PS: I suggest using a capital `-V` for the short option. That way the lowercase `-v` is still available for a potentially future option that will be used more frequently (e.g., `--verbose`) than the version switch.

Cheers.

**Comment by bwhmather** [comment]
I can't think of any reason why not.  Fixed in https://github.com/bwhmather/ssort/pull/111.

Cheers.

**Comment by bwhmather** [comment]
(published in 0.12.3)
